project/Settings.scala

```diff
@@ -98,13 +99,16 @@ object Settings {
     dockerUpdateLatest := true,
     dockerCommands ++= List(
       ExecCmd("CMD", "export", "CLASSPATH=lib/*jar")
+//      ExecCmd("CMD", "unmask", "umask g+w") //TODO: doesn't seem to do anything yet
     )
   )
 
   lazy val serverDockerSettings = commonDockerSettings ++ List(
     mainClass in Compile := Some("org.broadinstitute.dsp.workbench.welder.server.Main"),
     packageName in Docker := "broad-dsp-gcr-public/welder-server",
-//    daemonUser in Docker := "welder-user",
+    daemonUser in Docker := "welder-user",
```

Comment:
Please revert so you do not accidentally merge this

Reply:
My PR (https://github.com/DataBiosphere/welder/pull/23) updates this, can probably revert all `Settings.scala` changes here.

Reply:
This will be taken care of after rebasing